src/bin/console/console.cpp

```diff
@@ -115,6 +115,8 @@ Console::Console()
   for (auto it = tpcc_generators.begin(); it != tpcc_generators.end(); ++it) {
     _tpcc_commands.push_back(it->first);
   }
+
+  _prepared_statement_cache = std::make_shared<SQLQueryCache<SQLQueryPlan>>(DefaultCacheCapacity);
```

Comment:
Not sure if we should call this a "cache" (where entries get evicted) or a store.